### ISSUE TITLE
Ink Annotation vertices does not return vertices

### ISSUE THREAD

**Issue by PowerSnail** [pymupdf/PyMuPDF#525]
_**Please provide all mandatory information!**_

## Describe the bug (mandatory)

When annotation type is `Ink`, `.vertices` return `[(0.0, 842.0)]`.

## To Reproduce (mandatory)

```python
import fitz

pdf = fitz.Document(filetype="pdf")
page = pdf.newPage()
page.addInkAnnot([[(300,300), (400, 380), (350, 350)]])
for a in page.annots():
    print(a.vertices)
pdf.save("test.pdf")  # Check the output PDF that the annotation is correctly drawn
```

The output is always `[(0.0, 842.0)]` regardless of what the input is.

## Expected behavior (optional)

Per description on the [documentation](https://pymupdf.readthedocs.io/en/latest/annot/#Annot.vertices), the vertices should be a list of list of points.

## Your configuration (mandatory)

```
>>> print(sys.version, "\n", sys.platform, "\n", fitz.__doc__)

3.8.3 (tags/v3.8.3:6f8c832, May 13 2020, 22:37:02) [MSC v.1924 64 bit (AMD64)] 
 win32

PyMuPDF 1.17.0: Python bindings for the MuPDF 1.17.0 library.
Version date: 2020-05-13 20:05:13.
Built for Python 3.8 on win32 (64-bit).
```

**Comment by JorjMcKie** [comment]
Issue confirmed! Thanks for reporting this!

**Comment by JorjMcKie** [comment]
Resolved by v1.17.1 which is currently being uploaded to PyPI.
